template/partial/safety_plan_worksheet/concern_1.html

```diff
@@ -0,0 +1,16 @@
+{% set ribbon_data = { color: 'red', text: 'Physical Safety: Concerns'} %}
+{% include 'partial/ribbon.html' with ribbon_data only %}
+<p>Have you anticipated where and in which ways you might come into contact with the perpetrator and perpetrator’s friends/family? If you were to come into contact with the perpetrator and perpetrator’s family/friends, what specific things could you do that might help you feel safe in that situation?
```

Comment:
Can you indent the tags one below another with 4 spaces as you have done for other html files?